Installation/changes.html

```diff
@@ -230,6 +230,14 @@ <h2 id="release4.7">Release 4.7 </h2>
     </li>
   </ul>
 
+  
+  <h3>Polyhedron Demo (Scale Space Reconstruction Plugin)</h3>
+  <ul>
+    <li>Speed up construction of polygon soups by only inserting the
+      vertices that are actually used by each polygon soup.
+    </li>
+  </ul>
+
 
```

Comment:
Le 30/06/2015 11:24, Laurent Rineau a écrit :
>
> In Installation/changes.html 
> <https://github.com/CGAL/cgal/pull/144#discussion_r33554818>:
>
> > @@ -230,6 +230,14 @@ <h2 id="release4.7">Release 4.7 </h2>
> >      </li>
> >    </ul>
> >
> > +
> > +  <h3>Polyhedron Demo (Scale Space Reconstruction Plugin)</h3>
> > +  <ul>
> > +    <li>Speed up construction of polygon soups by only inserting the
> > +      vertices that are actually used by each polygon soup.
> > +    </li>
> > +  </ul>
> > +
> >
>
> Do we ask Simon to remove the modification to changes.html, or it is 
> acceptable that way?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/CGAL/cgal/pull/144/files#r33554818>.
>
I cancelled the modifications in changes.html this morning before 
pushing it into integration.

-- 
Simon Giraudot